diesel_tests/Cargo.toml

```diff
@@ -31,6 +31,10 @@ stable_sqlite = ["with-syntex", "sqlite", "diesel_codegen_syntex/sqlite"]
 unstable_postgres = ["unstable", "postgres", "diesel_codegen/postgres"]
 unstable_sqlite = ["unstable", "sqlite", "diesel_codegen/sqlite"]
 
+[lib]
+name = "integration_tests"
+path = "tests/lib.rs"
+
```

Comment:
Oh whoops that wasn't supposed to get committed. I do that when I need to do `-Ztrace-macros`